pkgs/applications/video/manim/failing_tests.nix

```diff
@@ -64,6 +64,16 @@
   "test_PointCloudDot"
   "test_Torus"
 
+  # test_ImplicitFunction[/test_implicit_graph] failing with:
+  # E           AssertionError:
+  # E           Not equal to tolerance rtol=1e-07, atol=1.01
+  # E           Frame no -1. You can use --show_diff to visually show the difference.
+  # E           Mismatched elements: 1185[/633] / 1639680[/1639680] (0.0723[/0.0386]%)
+  # E           Max absolute difference: 125[/121]
+  # E           Max relative difference: 6.5[/1]
+  "test_ImplicitFunction"
```

Comment:
Is there any indication why these a newly failing 🤔

Reply:
I don't really know, but I think that one of the changed dependencies had a breaking change in how they calculated tolerances. I _hope_ that we just get away with ignoring it, considering that the pr, that fixes should fix that upstream, is already merged (and we just have to wait for the next release).

Reply:
Maybe @friedelino might have a idea about these tests (as they are the manim maintainer).

Reply:
Not ideal but looks like in the same vein as
   "test_Text2Color"
    "test_PointCloudDot"
    "test_Torus"
above

 I'd be OK with this as long as you leave a comment such that future maintainers can try to shrink failing_tests (e.g. Failure likely due to tolerance difference, consider removing in future updates as dependencies synchronize)

Reply:
Good point! I've also included a link to the pr, to give a bit of context.